src/rest-server/src/utils/manager/group/group.js

```diff
@@ -29,13 +29,23 @@ const groupSchema = Joi.object().keys({
     .default(''),
   extension: Joi.object()
     .pattern(/\w+/, Joi.required())
-    .required(),
+    .keys({
+      acls: Joi.object()
```

Comment:
From the perspective of OpenPAI, I think this change is fine, because we could strictly ask each opertion add the acl filed to every group, or the operation will fail.

Reply:
But, I wonder if it is possible to remove this check into model level? Or in the next time, if we add new value into extension, we will have to modify the schema and add new entry into it again. Or do you have a better solution?

Reply:
We already have two validation and defaulting: input from route, output to k8s. I'm not sure it's a good idea to introduce more default logic at other level. I think setup all defaulting at the bottom could make upper level more clean
For compatible schema change, I think it's fine, especially we are changing extension field, which is designed for changable.

Reply:
pending design？